config.js

```diff
@@ -1,27 +1,25 @@
-"use strict";
```

Comment:
Maybe it was because of “SyntaxError: Block-scoped declarations (let, const, function, class) not yet supported outside strict mode”?